5.6/root/usr/share/container-scripts/php/common.sh

```diff
@@ -0,0 +1,140 @@
+}
+
+function log_info {
+  echo "---> `date +%T`     $@"
+}
+
+function log_and_run {
+  log_info "Running $@"
+  "$@"
+}
+
+function log_volume_info {
+  CONTAINER_DEBUG=${CONTAINER_DEBUG:-}
+  if [[ "${CONTAINER_DEBUG,,}" != "true" ]]; then
+    return
+  fi
+
+  log_info "Volume info for $@:"
+  set +e
+  log_and_run mount
+  while [ $# -gt 0 ]; do
+    log_and_run ls -alZ $1
+    shift
+  done
+  set -e
+}
+
+# get_matched_files finds file for image extending
+function get_matched_files() {
+  local custom_dir default_dir
+  custom_dir="$1"
+  default_dir="$2"
+  files_matched="$3"
+  find "$default_dir" -maxdepth 1 -type f -name "$files_matched" -printf "%f\n"
+  [ -d "$custom_dir" ] && find "$custom_dir" -maxdepth 1 -type f -name "$files_matched" -printf "%f\n"
+}
+
+# process_extending_files process extending files in $1 and $2 directories
+# - source all *.sh files
+#   (if there are files with same name source only file from $1)
+function process_extending_files() {
+  local custom_dir default_dir
+  custom_dir=$1
+  default_dir=$2
+
+  while read filename ; do
+    echo "=> sourcing $filename ..."
+    # Custom file is prefered
+    if [ -f $custom_dir/$filename ]; then
+      source $custom_dir/$filename
+    elif [ -f $default_dir/$filename ]; then
+      source $default_dir/$filename
+    fi
+  done <<<"$(get_matched_files "$custom_dir" "$default_dir" '*.sh' | sort -u)"
+}
+
+# process extending config files in $1 and $2 directories
+# - expand variables in *.conf and copy the files into /opt/app-root/etc/httpd.d directory
+#   (if there are files with same name source only file from $1)
+function process_extending_config_files() {
+  local custom_dir default_dir
+  custom_dir=$1
+  default_dir=$2
+
+  while read filename ; do
+    echo "=> sourcing $filename ..."
+    # Custom file is prefered
+    if [ -f $custom_dir/$filename ]; then
+       envsubst < $custom_dir/$filename > ${HTTPD_CONFIGURATION_PATH}/$filename
+    elif [ -f $default_dir/$filename ]; then
+       envsubst < $default_dir/$filename > ${HTTPD_CONFIGURATION_PATH}/$filename
+    fi
+  done <<<"$(get_matched_files "$custom_dir" "$default_dir" '*.conf' | sort -u)"
+}
+
+# Copy config files from application to the location where httpd expects them
+# Param sets the directory where to look for files
+# This function was taken from httpd container
+process_config_files() {
+  local dir=${1:-.}
+  if [ -d ${dir}/httpd-cfg ]; then
+    echo "---> Copying httpd configuration files..."
+    if [ "$(ls -A ${dir}/httpd-cfg/*.conf)" ]; then
+      cp -v ${dir}/httpd-cfg/*.conf "${HTTPD_CONFIGURATION_PATH}"/
+      rm -rf ${dir}/httpd-cfg
+    fi
+  fi
+}
+
+# Copy SSL files provided in application source
+# This function was taken from httpd container
+process_ssl_certs() {
+  local dir=${1:-.}
+  if [ -d ${dir}/httpd-ssl/private ] && [ -d ${dir}/httpd-ssl/certs ]; then
+    echo "---> Looking for SSL certs for httpd..."
+    cp -r ${dir}/httpd-ssl ${HTTPD_APP_ROOT}
```

Comment:
`HTTPD_APP_ROOT` is missing. (cause of CI failure)

Reply:
Thx, it's was a copy/paste issue, fixing it just now.